src/work-allocation/components/task-manager-filter/task-manager-filter.component.ts

```diff
@@ -83,7 +83,7 @@ export class TaskManagerFilterComponent implements OnInit, OnDestroy {
       enableCondition: 'selectLocation=search',
       minSelectedError: 'You must select a location',
       maxSelectedError: null,
-      enableAddLocationButton: false,
+      enableAddButton: false,
```

Comment:
Regarding this, has there been a change in common-lib regarding this because the config is determined by that?

Reply:
Yes, there has been a change